src/pytorch_lightning/utilities/xla_device.py

```diff
@@ -11,90 +11,36 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-import functools
-import os
-import queue as q
-import traceback
-from multiprocessing import Process, Queue
-from typing import Any, Callable, Union
 
-from pytorch_lightning.utilities.imports import _XLA_AVAILABLE
+from multiprocessing import Queue
+from typing import Any, Callable
 
-if _XLA_AVAILABLE:
-    import torch_xla.core.xla_model as xm
-
-# define TPU availability timeout in seconds
-TPU_CHECK_TIMEOUT = 60
```

Comment:
and that one :D

Reply:
https://github.com/Lightning-AI/lightning/pull/14514